### PR TITLE
Make camel-quarkus-rest depend on camel-quarkus-platform-http

### LINKED ISSUE #1244: Make camel-quarkus-rest depend on camel-quarkus-platform-http
We could make the camel-quarkus-rest to depend on camel-quarkus-platform-http as that is the recommended set-up we do recommend.

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #1244

[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: integration-tests/rest/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+#
+# Quarkus
+#
+quarkus.log.file.enable = false
```
Comment (ppalaga) [inline integration-tests/rest/src/main/resources/application.properties:21]: `false` is the default BTW https://quarkus.io/guides/all-config#quarkus-core_quarkus.log.file

Comment (lburgazzoli) [reply integration-tests/rest/src/main/resources/application.properties]: removed

**Comment by lburgazzoli** [comment]
> One nitpick inline. Besides, this should be documented, perhaps on the https://camel.apache.org/camel-quarkus/latest/extensions/rest.html page?

done

**Comment by lburgazzoli** [comment]
Requires https://issues.apache.org/jira/browse/CAMEL-15109, closing for now

**Pull request closed without merging**